cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -362,7 +362,11 @@ func (s *CMServer) Run(_ []string) error {
 				metrics.DefaultHeapsterService,
 				metrics.DefaultHeapsterPort,
 			)
-			podautoscaler.NewHorizontalController(hpaClient, metricsClient).
+			// TODO parameterize tolerance/downscale/upscale options.
+			tolerance := 1.0
```

Comment:
@wojtek-t @jszczepkowski  this is the culprit.  I made the default tolerance was way to high (off by a decimal)!  I will fix and re-validate this manually.  which were the specific autoscale tests that it broke?  just for good measure ill paste results in the PR ...